src/gfloat/round.py

```diff
@@ -103,11 +88,20 @@ def round_float(fi: FormatInfo, v: float, rnd=RoundMode.TiesToEven) -> float:
             return 0.0
 
     # Overflow
-    if result > fi.max:
-        if fi.has_infs:
-            result = np.inf
-        else:
+    if rnd == RoundMode.OCP_NONSAT:
+        if v > fi.max:
```

Comment:
I think `v` here should be `result` as the SAT / NONSAT happens to the rounded value.

Reply:
Added a comment

Reply:
Argh, now I see that the table says "after rounding".  Will fix.

Reply:
My misunderstanding - I had somehow decided that OCP SAT/NONSAT happened to the raw value.